src/NativeScript/ObjC/Unmanaged/UnmanagedType.h

```diff
@@ -0,0 +1,54 @@
+#ifndef UnmanagedType_h
+#define UnmanagedType_h
+
+#include <stdio.h>
+#include "FFIType.h"
+#include "UnmanagedPrototype.h"
+#include "UnmanagedInstance.h"
+
+#endif /* UnmanagedType_h */
+
+namespace NativeScript {
+class UnmanagedConstructor : public JSC::JSDestructibleObject {
+public:
+    typedef JSC::JSDestructibleObject Base;
+
+    static UnmanagedConstructor* create(JSC::VM& vm, JSC::JSGlobalObject* globalObject, JSC::JSCell* returnType, JSC::Structure* structure) {
+        UnmanagedConstructor* constructor = new (NotNull, JSC::allocateCell<UnmanagedConstructor>(vm.heap)) UnmanagedConstructor(vm, structure);
+        constructor->finishCreation(vm, globalObject, returnType);
+        return constructor;
+    }
+
+    DECLARE_INFO;
+
+    static JSC::Structure* createStructure(JSC::VM& vm, JSC::JSGlobalObject* globalObject, JSC::JSValue prototype) {
+        return JSC::Structure::create(vm, globalObject, prototype, JSC::TypeInfo(JSC::ObjectType, StructureFlags), info());
+    }
+
+    const FFITypeMethodTable& ffiTypeMethodTable() const {
+        return this->_ffiTypeMethodTable;
+    }
+
+private:
+    JSC::WriteBarrier<JSC::JSCell> _returnType;
```

Comment:
There isn't a specific order at the moment, but grouping fields together would make it easier for searching.